client/src/pages/home/Homepage.js

```diff
@@ -1,12 +1,38 @@
+import { useLocation } from "react-router-dom";
+import { useRef, useEffect } from "react";
 import About from "./components/about/About";
 import ContactUs from "./components/contact-us/ContactUs";
 
 function Home() {
+	const about = useRef(null);
+	const contactUs = useRef(null);
```

Comment:
Variables name must be clear about purpose in order for readability. In this case, I will name it `aboutRef` to give other programmers a sense it use Ref hook to reference about section.